docs/concepts/fs/feature_group/feature_pipelines.md

```diff
@@ -32,10 +32,14 @@ Spark is popular as a feature engineering framework as it can scale to process l
 
 SQL has grown in popularity for performing heavy lifting in feature pipelines - computing aggregates on data - when the input data already resides in a data warehouse. Data warehouses also support data validation, for example, through Great Expectations in DBT. However, SQL is not mature as a platform for transformations and dimensionality reductions, where UDFs are applied row-wise.
 
-
 You can do aggregation in SQL for data in your data warehouse or database.
 
+
 ### Feature Engineering in Flink
+Apache Flink is a powerful and flexible framework for stateful feature computation operations over unbounded and bounded data streams. It is used for feature engineering when you need very fresh features computed in real-time. Flink provides a rich set of operators and functions such as time windows and aggregation operations that can be applied to keyed and/or global window streams. Flink’s stateful operations allows it to maintain and update state across multiple data records or events, which is particularly useful for feature engineering tasks such as sessionization and/or maintaining rolling aggregates over a sliding window of data.
```

Comment:
`Flink’s stateful operations allows it to maintain` -> `Flink’s stateful operations allow users to maintain`